provision/aws/rosa_cluster_reaper.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/sh
+set -e
+
+if [ -f ./.env ]; then
+  source ./.env
+fi
+
+echo "Starting the ROSA cluster reaper at $(date -uIs)"
+
+# Get a list of running ROSA clusters
+CLUSTERS=$(rosa list clusters --output json | jq -r '.[] | select(.state == "ready") | .name')
+
+if [ -z "$CLUSTERS" ]; then echo "Didn't find any ready state clusters"; exit 1; fi
```

Comment:
Should we not also destroy any clusters not in ready state? I.e. those that are installing or have failed for some reason?